terragrunt/aft/main/Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: fmt docs test
+
+fmt:
+	@terraform fmt -recursive
+
+docs:
+	@rm -rf .terraform
+	@rm -f .terraform.lock.hcl
```

Comment:
Could you please remove these two lines from the Makefiles as well.

Reply:
Removed for all files - thank you for flagging this.